molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/.yamllint

```diff
@@ -7,7 +7,8 @@ rules:
   brackets:
     max-spaces-inside: 1
     level: error
-  line-length: disable
```

Comment:
This change conflicts with https://github.com/ansible/molecule/pull/1659.

You could potentially drop removing the comment to allow #1659 to do that (along with other needed docs fixes).

Reply:
done